src/components/landing/GetStartedButton.tsx

```diff
@@ -1,15 +1,23 @@
+import { displayFromSm, displayFromXsToSm } from 'helpers/visibilityClassnames'
 import { useNavigate } from 'react-router-dom'
 import Button from 'components/Button'
 import ChildrenProp from 'models/ChildrenProp'
-import useBreakpoints from 'hooks/useBreakpoints'
 
 export default function ({ children = 'Get started' }: ChildrenProp) {
   const navigate = useNavigate()
-  const { xs } = useBreakpoints()
 
   return (
-    <Button type="primary" small={xs} onClick={() => navigate('/app')}>
-      {children}
-    </Button>
+    <>
+      <span className={displayFromXsToSm}>
+        <Button type="primary" small onClick={() => navigate('/app')}>
```

Comment:
Maybe you can handle `small` inside the button? Maybe we can have a prop `smallFromXsToSm`? I think we should avoid repeating ourselves only to use tailwind breakpoints, in this case `useBreakpoint` saves us a few lines, so it's better to use it, I guess

Reply:
I think it's better to do like so:
1) If we need to pass the `xs` as prop - then let's use `useBreakpoints`
2) If we have two different components, then let's use tailwind

Reply:
Returned `useBreakpoints` back